pkg/porter/schema.go

```diff
@@ -130,15 +129,15 @@ func (p *Porter) injectMixinSchemas(ctx context.Context, manifestSchema jsonSche
 
 		for _, action := range coreActions {
 			actionItemSchema, ok := actionSchemas[action]["items"].(jsonSchema)
-			if err != nil && p.Debug {
-				fmt.Fprintln(p.Err, errors.Errorf("root porter manifest schema has invalid properties.%s.items type, expected map[string]interface{} but got %T", action, actionSchemas[string(action)]["items"]))
+			if !ok && p.Debug {
+				fmt.Fprintln(p.Err, fmt.Errorf("root porter manifest schema has invalid properties.%s.items type, expected map[string]interface{} but got %T", action, actionSchemas[string(action)]["items"]))
```

Comment:
🛑 ⚠️ 👀 Imo (but maybe I'm wrong because I dont know the logic very well), there was an issue here. The code was 
```
actionItemSchema, ok := actionSchemas[action]["items"].(jsonSchema)
if err != nil && p.Debug {
    ...
}
```

I think we needed to use `ok` instead of `err`. Because previously, `ok` was never used nor checked. If it wasn't a mistake, we should replace `ok` by `_`.